content/learning-paths/laptops-and-desktops/windowsperf_wpa_plugin/windowsperf_wpa_plugin.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: learningpathall
+title: WindowsPerf WPA Plugin
+weight: 2
+---
+
+# Overview
+
+[**WindowsPerf**](https://github.com/arm-developer-tools/windowsperf) is a lightweight performance profiling tool inspired by Linux perf, specifically tailored for Windows on Arm.
+It leverages the Arm64 PMU (Performance Monitor Unit) and its hardware counters to offer precise profiling capabilities.
+The **WindowsPerf WPA plugin** bridges the gap between the detailed output of **WindowsPerf** and the powerful capabilities of **Windows Performance Analyzer**.
+
+## WindowsPerf WPA Plugin releases
+
+You can find all binary releases of `WindowsPerf WPA Plugin` [here](https://github.com/arm-developer-tools/windowsperf-wpa-plugin/releases).
+
+# Installation
+
+For installation instructions see the [install guide](/install-guides/windows-perf-wpa-plugin).
+
+## Using WindowsPerf WPA Plugin
+
+In order to use the `WindowsPerf WPA Plugin`, we first need to get a `.json` output from a `wperf stat` command running on a Windows on Arm machine.
```

Comment:
```suggestion
In order to use the `WindowsPerf WPA Plugin`, we first need to get a `.json` output from a WindowsPerf `wperf stat` command running on a Windows on Arm machine.
```

Reply:
Also:

- You can add explanation that `wperf` command line interface `--json` command line option is used to force JSON output. 
- Please also add information that WPA-plugin opens JSON file.
- And maybe a window showing how to open JSON file.

In general it would be good to have and example where we use `--output example_1.json` and how we:
1. Open WPA
2. Select plugin and
3. open JSON file.